app/views/twilio/call.haml

```diff
@@ -1,3 +1,3 @@
 !!! XML
-%Response
-  %Dial= @user.phone
+%response
```

Comment:
No, no, I think if I'd given this file a proper extension, perhaps haml lint wouldn't have intervened.